src/extract.ts

```diff
@@ -304,6 +304,69 @@ function extractGoogleCppResult(output: string): BenchmarkResult[] {
     });
 }
 
+function extractCatch2Result(output: string): BenchmarkResult[] {
+    const lines = output.split('\n');
+
+    const ret = [];
+    // Example:
+    //   test bench_fib_20 ... bench:      37,174 ns/iter (+/- 7,527)
```

Comment:
Would you replace this line with the output from Catch2 framework?

Reply:
Changed, let me know if stuff is still missing